woocommerce/woocommerce-gateway-stripe
# Replace branded button instead of appending it to avoid duplication

If the payment button is enabled on checkout, the branded button is duplicated every time checkout is re-calculated. This occurs because the button is being appended to the wrapper, instead of replacing its contents. This commit replaces the wrapper contents instead of appending the button, so it's not duplicated.

Fixes #1220.

#### Changes proposed in this Pull Request:

- Replace branded button in wrapper instead of appending it

#### Testing instructions:

1. Add the following filter to add the Payment Request Button to the checkout page:
`add_filter( 'wc_stripe_show_payment_request_on_checkout', '__return_true' );`
2. Set the option 'Payment Request Button Type' to 'Branded' in plugin settings
3. Add an item to the cart and go to the Checkout page
4. The Google Pay button should appear once
5. Toggle a checkout recalculation by changing the billing/shipping country
6. The Google Pay button should appear once
